rbc/omnisci_backend/helper.py

```diff
@@ -0,0 +1,41 @@
+import functools
+from numba.core import extending
+
+
+ADDRESS = ("https://data-apis.org/array-api/latest/API_specification"
+           "/generated/signatures.{0}.{1}.html"
+           "#signatures.{0}.{1}")
+
+
+class Expose:
```

Comment:
If this file is just for the doc, I suggest renaming it so that it's clear that it's for the doc. And also "mark" the file as private with a leading `_`.